examples/pykokkos/from_array.py

```diff
@@ -16,17 +16,23 @@ def addition_cp(i: int, cp_arr: pk.View2D[int]):
 
 np_arr = np.zeros((size, 2)).astype(np.int32)
 cp_arr = cp.zeros((size, 2)).astype(np.int32)
+list_arr = [np.array([0, 0], dtype=np.int32)] * size
```

Comment:
When you initialize a list of objects like this all the elements of the list will reference the same object. Is this what you wanted here?

Reply:
Made the workload slightly more interesting to clarify what each thread was doing. Looks like it figured it out!

Reply:
Because of the numpy translation in the backend, you have to pull the data out of the view manually to return to a list format but it does seem to just make N copies when doing this: [ <init_object>] * N